### PR TITLE
Typos and Video embed fixes

### PR DESCRIPTION
**Pull request by ganesh-sanap**
## Category

- [x] Content fix

## What's in this Pull Request?

Fixed typos and Video embed in article

### CODE REVIEW

File: sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Update-Module -Name Microsoft.Online.SharePoint.PowerShell
 
 To open the SharePoint Online Management Shell command prompt, from the **Start** screen, type **sharepoint**, and then click **SharePoint Online Management Shell**.
 
-[!VIDEO https://www.youtube.com/embed/TMzHAWEQjlk]
+[!Video https://www.youtube.com/embed/TMzHAWEQjlk]
```
Comment (get-itips) [inline sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md:32]: I am not 100% sure but I think that the VIDEO tag is correct but needs a >
`> [!VIDEO https://...]` 

Example https://raw.githubusercontent.com/MicrosoftDocs/azure-docs/master/articles/azure-portal/azure-portal-video-series.md 

https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-video-series

Comment (get-itips) [reply sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md:32]: (Thanks for your contribution!)

**Comment by tseward** [comment]
Two things I would add to this:

* Show how to install the module without local admin rights (`-Scope CurrentUser`).
* Remove the variables in the examples; they're superfluous and increase the number of characters typed

File: sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md (hunk 1 above)
Comment (ganesh-sanap) [reply sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md]: I was referencing this: https://github.com/SharePoint/sp-dev-docs/edit/master/docs/spfx/set-up-your-developer-tenant.md and missed `>`.

Updated file with suggested changes, Thank you!

**Review by dariomws — APPROVED** [review]
I'm ok after suggested changes. Thanks!

**Comment by yogkumgit** [comment]
@ganesh-sanap

**Comment by ganesh-sanap** [comment]
> @ganesh-sanap

I have made changes suggested by @get-itips to "Video" tag in article. Do you want me to make changes suggested by @tseward?

**Comment by yogkumgit** [comment]
yes :)

**Comment by ganesh-sanap** [comment]
@yogkumgit Updated article with suggested changes, please review.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Pull request merged**